.circleci/config.yml

```diff
@@ -618,20 +633,19 @@ jobs:
             curl -o /tmp/pr https://api.github.com/repos/libra/libra/pulls/${pr_num}
             export BRANCH=$( cat /tmp/pr | jq ".base .ref" | sed 's/"//g' )
 
-            export IS_RELEASE=$( if [[ "$BRANCH" =~ "^v[0-9|.]+-release$" ]]; then echo true; else echo false; fi )
-            export IS_TEST=$( if [[ "$BRANCH" =~ "^testflow[0-9|.]+$" ]]; then echo true; else echo false; fi )
-            export IS_PRE_RELEASE=$( if [[ "$BRANCH" =~ "^premainnet-[0-9|.]+$" ]]; then echo true; else echo false; fi )
+            export IS_RELEASE=$( if [[ "$BRANCH" =~ "^release-[0-9|.]+$" ]]; then echo true; else echo false; fi )
```

Comment:
Are we changing the release branch from `v1-release` to `release-1`?

Reply:
@bmwill preferred this naming structure.   And it's easier for humans, so I was fine with changing to this.   Guess I need to track down the docs :D.

Reply:
I believe I said it was easier to read, but that it wasn't important or an issue either way. You jumped on changing it :)